PW40_2024_GranCanaria/Projects/Template/README.md.j2

```diff
@@ -54,3 +54,9 @@ key_investigators:
      If possible, also add links to sample data, and to any relevant publications. -->
 
 {{ background }}
+
+# Discussion
+
+<!-- Include here a link to a GitHub Issue or other place for discussing this project. -->
+
+* [Project Issue]({{ discussion_url }})
```

Comment:
```suggestion
* [Project Issue]({{ linked_issue_html_url }})
```

Reply:
Should we also include in this list the name of the discord channel ? Or at least reference that one may be created ...

```
* Project-specific discussion may also take place in our [Discord](../../#discord).
```

Not sure about the relative link to discord ...

Reply:
I would skip the discord link.  There is not guaranteed to be a discord channel for each project (since we are planning to try to conserves channels due to the 500 cap).